src/worker/storage.ts

```diff
@@ -245,6 +246,14 @@ export default class Storage {
     }
     return kmpFile.package;
   }
+
+  async exportProjectData(): Promise<Blob> {
+    return await exportDB(this.db);
+  }
+
+  async importProjectData(data: Blob): Promise<void> {
+    await importInto(this.db, data, { overwriteValues: true });
```

Comment:
So these methods export the ENTIRE database — not just the current project. Since the entire DB != one language project, this is doing too much.

Reply:
Ok got it, I'll make these changes